docs/scarpet/api/Auxiliary.md

```diff
@@ -191,6 +191,9 @@ produce an exception.
 
 Displays the result of the expression to the chat. Overrides default `scarpet` behaviour of sending everyting to stderr.
 
+### `action_bar(player, message)`, `action_bar(player_list,message)`
```

Comment:
This feels redundant, especially that you already expect the player. `print` aready takes optional player - make it a third argument for print
`print(message)`, `print(player, message)`, `print(target, player, message`
in this case target could be 'chat' (default), 'title', 'subtitle', 'action_bar' etc (isn't action bar a subtitle)

This way you could also add a player array support to print.

Reply:
ok boss.

Reply:
I personally see titles (and subtitles and actionbar) different from chat, and think that they should have a different call, since, as opposed to chat, titles can have their fade and stay times customized, feature that print can't provide (at least in its current state).

Reply:
if that can be customized (don't know, didn't play with datapacks much before), then it makes sense to provide a separate endpoint for them, leaving print just doing chat.

Reply:
ye I can't find the method for sending titles to the player...

Reply:
Ill resend to screen.

Reply:
Illl just add it to print with a switch statement so I can expand when I find method to send a title.

Reply:
The way you change title times in vanilla (don't know about datapacks) is by first running `/title times [in] [stay] [out]`, then run regular `/title`. It's quite tedious needing to run 2 commands for one action, but it's what it has.